### ISSUE TITLE
Kriya and process implications: Versioning and history information and dates

### ISSUE THREAD

**Issue by Melissa37** [elifesciences/XML-mapping#26]
See https://github.com/elifesciences/XML-mapping/commit/0bc1262ac4bf563d2446e0cf69738533bf6cf852

This new requirement has big implications for the workflow.
- Exeter will need to know of ALL previous versions of PoA content and add them to the XML (dates of publication and URL links)
- Kriya will require the addition of two new fields - publication date for PoA'd content (add the new VoR date)
- Kriya will need a new field so production can add a note if publishing a new version of a VoR so they can indicate the changes made

**Comment by gnott** [comment]
These dates could be all available from Lax I think. Is it possible ExeterPremedia can consume that data directly?

We could also look at whether this can be added automatically to the XML by the bot when it is processed.

Can we discuss on our next call agenda?

**Comment by Melissa37** [comment]
Yes lets chat about it on Tuesday - I did think option 1 would be easiest, but the need to add comments to versions would probably prevent that.

The main problem for Exeter would be PoA versions and picking up versions of PoA, and also the comments production would add.

This is also an issue for the PoA workflow too - if we do a V2 adding a comment about the change.
